packages/cli/templates/vercel/image-loader.mjs

```diff
@@ -0,0 +1,21 @@
+import { imageBaseUrl } from "~/constants.mjs";
+
+/**
+ * @type {import("@webstudio-is/image").ImageLoader}
+ */
+
+export const imageLoader = ({ quality, src, width }) => {
```

Comment:
Why split into separate module? Might be harder to maintain.

Reply:
The `image-loader` is different for deployment targets. The loader for `vercel` is different from netlify and others. So, just using extracted it. So, there can be a default and one that is custom to target 👍 . We can still do it with `constants.mjs` But that moves around a cop of `assetsBase` and other constants everytime so

Reply:
It's not like assetsBase take a lot of logic to implement. I'd still have them in one place.

Reply:
ok, i just followed the suggestion, not to repeat stuff in multiple palaces like `deps` in `package.json` thing. Will change it back then 👍